dotnet/samples/Concepts/Memory/VectorStore_VectorSearch_Paging.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.SemanticKernel.Data;
+
+namespace Memory;
+
+/// <summary>
+/// An example showing how to do paging when there are many records in the database and you want to page through these page by page.
+///
+/// The example shows the following steps:
+/// 1. Create a Volatile Vector Store.
+/// 2. Generate and add some test data entries.
+/// 3. Read the data back using vector search by paging through the results page by page.
+/// </summary>
+public class VectorStore_VectorSearch_Paging(ITestOutputHelper output) : BaseTest(output)
+{
+    [Fact]
+    public async Task ExampleAsync()
```

Comment:
Prefer longer for descriptive method names as this will help if we add another Fact to this class